ChristianMDCH/ChristianMDCH.md

```diff
@@ -0,0 +1 @@
+ChristianMDCH Christian Hernandez
```

Comment:
The exercise is not completed
please review the code

Reply:
Code has been checked. Please review again